useStore.js

```diff
@@ -117,6 +119,33 @@ export const useListItemsStore = create(
           listItems: newSortedListItems,
         });
       },
+      updateCheck: (listItemId) => {
```

Comment:
this is basically a toggle function. You should name it like that. I had no idea what this function is doing when I read it earlier.